build-static-glibc.sh

```diff
@@ -0,0 +1,236 @@
+﻿#!/bin/bash
```

Comment:
Can we try to no duplicate the script but just add a switch on the existing one?

Reply:
That was what I thought of doing first, but it quickly became messy. I'll get it working with a docker container first and then see to combine the two scripts.